server/src/main/java/model/shooting/board/ShootingBoard.java

```diff
@@ -0,0 +1,138 @@
+package model.shooting.board;
+
+import model.shooting.field.HitResult;
+import model.shooting.ship.ShootingShip;
+import model.shooting.field.Field;
+import model.shooting.field.FieldStateName;
+import model.Coordinates;
+import model.placement.fleet.Fleet;
+import model.placement.ship.PlacedShip;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.HashSet;
+
+public class ShootingBoard implements ObservableBoard {
+    private static final int COLUMNS_NUMBER = 10;
+    private static final int ROWS_NUMBER = 10;
+    private Map<Coordinates, Field> coordinatesFieldMap;
```

Comment:
Wrapper, with just collection field, nothing else, and really basic gettter/logic